ts/store/reducers/entities/payments.ts

```diff
@@ -64,3 +64,8 @@ export const paymentByRptIdReducer = (
 
 export const paymentsByRptIdSelector = (state: GlobalState) =>
   state.entities.paymentByRptId;
+
+// to keep solid code encapsulation
+export const testablepaymentByRptIdReducer = isTestEnv
+  ? { initial_state: INITIAL_STATE }
+  : undefined;
```

Comment:
In this case `INITIAL_STATE` is just a `const` and not a `function` that can modify an existing state.  I can't think of any side effects, what you think about export the `const` directly?

Reply:
indeed
it can't be dangerous
https://github.com/pagopa/io-app/pull/2736/commits/1ff8f2272cae3caa9580ac6401540a142d6d5233